src/lib/tradingView/utils.ts

```diff
@@ -1,6 +1,8 @@
+import { OrderSide } from '@dydxprotocol/v4-client-js';
+
 import { Candle, TradingViewBar, TradingViewSymbol } from '@/constants/candles';
 
-import type { AppTheme, AppColorMode } from '@/state/configs';
+import { AppTheme, type AppColorMode } from '@/state/configs';
```

Comment:
nit: import order

Reply:
addressed [here](https://github.com/dydxprotocol/v4-web/pull/292/files#diff-0ac79d7c49f51fabc55ba88f67c708c5b2c09600cdb2fe7227dced370842592bR6) in stacked PR